templates/overrides/search/block--facets-form.html.twig

```diff
@@ -0,0 +1,52 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a block.
+ *
+ * Available variables:
+ * - plugin_id: The ID of the block implementation.
+ * - label: The configured label of the block if visible.
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main content
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{% block content %}
+  {{ pattern('offcanvas', {
+    'title': {
+      'label': label,
+      'heading': 4,
+      'extra_classes': "mb-lg-4",
+     },
+    'attributes': attributes.addClass('bcl-offcanvas'),
+    'offcanvas_id': 'bcl-offcanvas',
+    'body': content,
```

Comment:
check if we can override content.form.actions.reset and use the link (or button?) pattern, if not we need to preprocess the block, worst case we form alter and theme the reset from there